scripts/release/update-dependencies-versions.js

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+
+/**
+ * @license Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* eslint-env node */
+
+'use strict';
+
+const path = require( 'path' );
+const { tools, logger } = require( '@ckeditor/ckeditor5-dev-utils' );
+
+const log = logger();
+const rootPath = path.join( __dirname, '..', '..' );
+const packageJsonPath = path.join( rootPath, 'package.json' );
+
+const mgitJson = require( path.join( rootPath, 'mgit.json' ) );
```

Comment:
I don't like basing this script on `mgit.json`. Mgit provides us infrastructure. It makes sure that subrepos are available. What if one of packages will be defined somewhere else? It's theoretical, but it shows that this is not the right starting point. Let's go through all dependencies of `ckeditor5/package.json` and simply get their versions by `require( packageName + '/package.json' ).version`.